rust/private/clippy.bzl

```diff
@@ -146,6 +160,10 @@ rust_clippy_aspect = aspect(
     fragments = ["cpp"],
     host_fragments = ["cpp"],
     attrs = {
+        "_capture_output": attr.label(
+            doc = "If true, write clippy output as a build output, rather than printing it, using the configured error_format",
```

Comment:
Nit: let's not duplicate the docs of the setting, we could say "Value of the `capture_clippy_output` setting" or something like that?